prometheus/prometheus.yml

```diff
@@ -7,9 +7,12 @@ remote_write:
     authorization:
       credentials: $PROM_REMOTE_WRITE_TOKEN
     write_relabel_configs:
+      - source_labels: [job]
```

Comment:
Q: how will we be able to separate out these metrics from each operator?

Reply:
Hmm this is a valid concern, since validator-ejector doesn't have identifiable info like charon does...

We can do relabling magic though, but it would require operators to edit this file in some way...